.azure-pipelines/config.yml

```diff
@@ -145,6 +145,35 @@ jobs:
       pathtoPublish: '$(Build.ArtifactStagingDirectory)'
       artifactName: 'bundle'
 
+- job: 'win_docker_build'
+  pool:
+    vmImage: 'windows-2019'
+  steps:
+  - template: 'templates/changes-include.yml'
+    parameters:
+      paths: 'go.mod go.sum Dockerfile.windows .azure-pipelines'
+  - bash: |
+      set -euxo pipefail
+
+      if [[ "$BUILD_SOURCEBRANCH" =~ ^refs/tags/v\d+\.\d+\.\d+$ ]]; then
+          AGENT_VERSION="${BUILD_SOURCEBRANCHNAME#v}"
+      else
+          AGENT_VERSION="latest"
+      fi
+
+      docker build -t quay.io/signalfx/signalfx-agent:${AGENT_VERSION}-windows --build-arg AGENT_VERSION=${AGENT_VERSION} -f Dockerfile.windows .
+
+      # push release image
+      if [[ "$AGENT_VERSION" != "latest" ]]; then
+          docker login -u="$QUAY_USERNAME" -p="$QUAY_TOKEN" quay.io
+          docker push quay.io/signalfx/signalfx-agent:${AGENT_VERSION}-windows
+      fi
+    displayName: 'Build windows docker image'
+    condition: and(ne(variables.hasChanges, 'false'), ne(variables.hasChanges, False))
```

Comment:
Just make it bool always and simplify this.

Reply:
The build has been taking over 20 minutes, so just wanted to limit unnecessary builds. But I can set to always build if desired.

Reply:
No I mean make it always `False` instead  of potentially `'false'` (string).  This condition is kind of irrelevant though if you want to build every time a release tag is pushed though.